src/content/docs/en/guides/imports.mdx

```diff
@@ -183,16 +183,17 @@ import logoUrl from '@assets/logo.png?url';
 These aliases are also integrated automatically into [VS Code](https://code.visualstudio.com/docs/languages/jsconfig) and other editors.
 
 
-## `Astro.glob()`
+## `import.meta.glob()`
```

Comment:
I guess there's more documentation on this page than I thought. What do you think about removing all of this except for the `#### Glob Patterns in Astro` section, which lists the limitations and links to the Vite docs?

Reply:
I'll shorten the section a bit, but we do prefer our own explanation for our own users. So, I think we'd prefer to rework this page, and in fact on all our other non reference pages, link HERE instead of Vite. The Vite page isn't the most user- friendly, so we'll link there from here and from reference. But in our guides, where people are trying to use it in their project and understand how it works, we'll link here. If they want more referency at that point, this will take them to Vite.